src/main/java/org/manifold/compiler/front/StaticAttributeAccessVertex.java

```diff
@@ -49,14 +43,18 @@ public Value getValue() {
     return this.value;
   }
 
+  protected abstract Value getVal(TupleValue tupleValue);
+  protected abstract String indexRepresentation();
```

Comment:
I think it's just supposed to be a generic name for "print how this field is going to be accessed".  But it should probably get a more general name then.